resources/js/components/Room/HistoryComponent.vue

```diff
@@ -0,0 +1,420 @@
+        </b-table>
+        <b-pagination
+          v-if="attendance.length>settings('pagination_page_size')"
+          v-model="attendanceCurrentPage"
+          :total-rows="attendance.length"
+          :per-page="settings('pagination_page_size')"
+          aria-controls='attendance-table'
+          align='center'
+        ></b-pagination>
+      </b-modal>
+
+    </b-overlay>
+  </div>
+</template>
+
+<script>
+import Base from '../../api/base';
+import LineChart from './../../charts/LineChart';
+import { mapGetters } from 'vuex';
+import RawText from '../RawText';
+
+export default {
+  components: { RawText, LineChart },
+  props: {
+    room: Object,
+    modalStatic: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  data () {
+    return {
+      // list of meetings
+      meetingsLoading: false,
+      meetingsLoadingError: false,
+      meetingsMeta: {
+        current_page: undefined,
+        total: undefined,
+        per_page: undefined
+      },
+
+      // statistics of meeting
+      statsLoading: false,
+      statsMeeting: null,
+      chartDataRows: {
+        participants: [],
+        voices: [],
+        videos: []
+      },
+
+      // attendance of meeting
+      attendanceLoading: false,
+      attendanceMeeting: null,
+      attendance: [],
+      attendanceCurrentPage: 1
+    };
+  },
+  methods: {
+    /**
+     * Loads the current and previous meetings of a given room and calls on finish the callback function.
+     *
+     * @param ctx Context information e.g. the current the page.
+     * @param callback
+     * @return {null}
+     */
+    fetchMeetings (ctx, callback) {
+      let data = [];
+
+      const config = {
+        params: {
+          page: ctx.currentPage
+        }
+      };
+
+      Base.call('rooms/' + this.room.id + '/meetings', config).then(response => {
+        this.meetingsMeta = response.data.meta;
+        data = response.data.data;
+        this.meetingsLoadingError = false;
+      }).catch(error => {
+        this.meetingsLoadingError = true;
+        Base.error(error, this.$root, error.message);
+      }).finally(() => {
+        callback(data);
+      });
+
+      return null;
+    },
+
+    /**
+     * Load statistics for given meeting
+     * @param meeting
+     */
+    loadMeetingStats (meeting) {
+      // enable loading indicator
+      this.statsLoading = true;
+      Base.call('meetings/' + meeting.id + '/stats')
+        .then(response => {
+          // set currentMeeting to the fetched meeting to show meeting start and end in modal
+          this.statsMeeting = meeting;
+
+          // parse statistical data to format that can be used by the computed property chartData
+          this.chartDataRows = {
+            participants: [],
+            voices: [],
+            videos: []
+          };
+          response.data.data.forEach(stat => {
+            const datetime = this.$date.utc(stat.created_at).tz(this.userTimezone).format('YYYY-MM-DD HH:mm');
```

Comment:
We should use the locale date format with [$d](https://kazupon.github.io/vue-i18n/guide/datetime.html)!

Reply:
For chartjs the datatime needs to be in this format. The display of the time is set in line 362